src/components/Post.jsx

```diff
@@ -426,7 +426,7 @@ function Post(prop) {
               </>
             ) : (
               <>
-                <p className="post_caption">{caption}</p>
+                <p style={{ whiteSpace: 'pre-line' }} className="post_caption">{caption}</p>
```

Comment:
instead of adding inline styles, can we put these styles where they belong in index.css?